src/main/webapp/resources/js/components/samples/components/SampleFileList.jsx

```diff
@@ -80,6 +85,29 @@ export function SampleFileList() {
       .then(({ data }) => {
         notification.success({ message: data.message });
         dispatch(removeFileObjectFromSample({ fileObjectId, type }));
+        if (
+          sample.defaultSequencingObject !== null &&
```

Comment:
Remember you can simplify this to `sample.defaultsequencingObject?.identifier === fileObjectId`

Reply:
Good catch. Updated in 4e8c30254a668636a8ec5217dd94db9910c1a389